ERCS/erc-4242.md

```diff
@@ -0,0 +1,106 @@
+---
+eip: 4242
```

Comment:
```suggestion
eip: 7648
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename and any references to use the issued number.